var/spack/repos/builtin/packages/kokkos/package.py

```diff
@@ -229,7 +236,9 @@ def setup_dependent_package(self, module, dependent_spec):
     def cmake_args(self):
         spec = self.spec
 
-        if spec.satisfies('~wrapper+cuda') and not spec.satisfies('%clang'):
+        if spec.satisfies("~wrapper+cuda") and not (
+            spec.satisfies("%clang") or spec.satisfies("%cce")
+        ):
```

Comment:
This is the formatting `black` gives me. Is this all right?

Reply:
I think black is not used, but it is being considered. fyi `spack style` should bootstrap whatever spack uses to validate style in github actions and then runs the checks.

Reply:
Yep, I discovered `spack style` today and it works fine! But it doesn't fix formatting as far as I can tell. I ran `spack style --fix` and I got the same complaints that CI gave, but formatting was unchanged. Did I miss something?

Reply:
I think only isort supports autofixing